conanfile.py

```diff
@@ -43,39 +44,39 @@ class UnitsConan(ConanFile):
     url = "https://github.com/mpusz/units"
     description = "Physical Units library for C++"
     exports = ["LICENSE.md"]
+    exports = ["*"]
```

Comment:
I am not sure if I copy all of the sources or just a `src` directory to Conan. End-user only cares about it and I do not want to trash his/her Conan cache with development source code (additional files, examples, unit tests, etc...). With `git` I did not have an option as it is hard to just clone a specific subdirectory and anyway I used top-level git repo for my CI of the library.